jbake-core/src/main/java/org/jbake/app/ContentStore.java

```diff
@@ -97,6 +105,7 @@ public final void updateSchema() {
     }
 
     public void close() {
+        db.activateOnCurrentThread();
```

Comment:
Please change this to call private method `activateOnCurrentThread()` to have a single entry point.